src/components/pages/typeform/TypeForm.js

```diff
@@ -2,17 +2,13 @@ import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 import * as typeformEmbed from '@typeform/embed'
 
-// TODO -> move to constants
-const TYPEFORM_URL = 'https://passculture.typeform.com/to/T8rurj'
+const { TYPEFORM_URL_CULTURAL_PRACTICES_POLL } = process.env
```

Comment:
On peux déplacer ça dans le fichier config et l'importer depuis la conf ?
```suggestion
import { TYPEFORM_URL_CULTURAL_PRACTICES_POLL } from '../../../utils/config'
```